pkg/secret/secret.go

```diff
@@ -0,0 +1,28 @@
+// Package contains interfaces for API secrets, storage and encoder.
+package secret
+
+//Provider defines what we can do with the data in our application
+//Describes the behavior of commands from the terminal
```

Comment:
This comment leaks unnecessary details of some terminal usage. I'll suggest to remove it and add something like "Gateway for managing data by setting/getting by a key."